src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopup.qml

```diff
@@ -30,14 +30,15 @@ Popup {
     property bool isOpened: false
     property bool opensUpward: false
 
-    property var arrowX: width / 2
+    property var arrowX: anchorItem != null ? anchorItem.x - x + anchorItem.width / 2 : width / 2
```

Comment:
Better to use Boolean(obj) instead of comparing with null:

arrowX: Boolean(anchorItem) ? ... : ...

Reply:
Fixed, thanks!